packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -708,18 +719,26 @@ export default {
             const logMessage = error.logMessage || 'Consumer Checkout Failure';
 
             this.$emit(eventToEmit, { ...this.eventData, error });
+
             this.logInvoker({
                 message: logMessage,
                 data: this.eventData,
                 logMethod: this.$logger.logError,
                 error
             });
+
             this.trackFormInteraction({ action: 'error', error: `error_${error.message}` });
-            this.genericErrorMessage = message;
 
-            this.$nextTick(() => {
-                this.scrollToElement(this.$refs.errorAlert.$el);
-            });
+            // We don't want to show a dialog and an error message.
+            // TODO: refactor `nonFulfillableError` and `genericErrorMessage` so that we try to use only one, and
```

Comment:
Can we move TODOs to tickets?

Reply:
Tickets are going to be created for this. But I think it's also good to keep the note there, so that if someone sees the code, they know it's going to change soon. Unless people generally prefer not to have this. Whatever the team prefers 😄

Reply:
If we have a ticket, could we reference it in here, otherwise there's the danger f it not getting removed after it's fixed and just being a confusing comment.

Reply:
I'm replying to you on Slack directly with regards to this.